src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -370,69 +401,81 @@ export default class PhotoField extends React.Component {
-        });
-        // wide images take the full height of the cropBox
-        // to center, uses the ratio of cropBoxHeight / cavasHeight
-      } else {
-        this.refs.cropper.setCanvasData({
-          height: cropBoxHeight,
+      // sometimes this callback is called before the new cropper has mounted
+      // check if the cropper instance has canvasData before setting its values
+      if (!_.isEmpty(this.refs.cropper.getCanvasData())) {
+        // center the cropbox using container width
+        const containerWidth = this.refs.cropper.getContainerData().width;
+        const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(this.props.formContext.pageTitle);
+        const left = smallScreen ? (containerWidth / 2) - 120 : (containerWidth / 2) - 150;
+        this.refs.cropper.setCropBoxData({
           top: 0,
-          left: (containerWidth - (cropBoxHeight / oldCanvasHeight * oldCanvasWidth)) / 2
+          left,
+          height: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE,
+          width: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE
         });
-      }
 
-      // with the canvas resized, use its dimensions to determine the min zoom ratio
-      const { width: newCanvasWidth } = this.refs.cropper.getCanvasData();
-      const minRatio = newCanvasWidth / naturalWidth;
-      const slider = this.refs.slider;
-      slider.value = minRatio;
+        const defaultCanvasData = getCanvasDataForDefaultPosition({
+          canvasData: this.refs.cropper.getCanvasData(),
+          cropBoxData: this.refs.cropper.getCropBoxData(),
+          containerWidth
+        });
 
-      this.setState({
-        zoomValue: minRatio,
-        minRatio
-      });
+        this.refs.cropper.setCanvasData(defaultCanvasData);
 
-      // update warning messages
-      const newCanvasData = getCanvasDataForMove({
-        canvasData: this.refs.cropper.getCanvasData(),
-        cropBoxData: this.refs.cropper.getCropBoxData(),
-      });
+        // with the canvas resized, use its dimensions to determine the min zoom ratio
+        const { width: newCanvasWidth, naturalWidth } = this.refs.cropper.getCanvasData();
+        const minRatio = newCanvasWidth / naturalWidth;
+        const slider = this.refs.slider;
+        slider.value = minRatio;
 
-      this.updateBoundaryWarningAndButtonStates(newCanvasData);
+        this.setState({
+          zoomValue: minRatio,
+          minRatio
+        });
+
+        // update warning messages
+        const boundedCanvasData = getBoundedCanvasPositionData({
+          canvasData: this.refs.cropper.getCanvasData(),
+          cropBoxData: this.refs.cropper.getCropBoxData(),
+        });
+
+        this.updateBoundaryWarningAndButtonStates(boundedCanvasData);
+      }
     });
   }
 
   maybeMoveCanvasWithinBounds = (moveData, zoomWarn = false) => {
-    const newCanvasData = getCanvasDataForMove({
+    // rotate
+    if (moveData && 'moveDegree' in moveData) {
```

Comment:
Added `moveDegree`. If this value is provided then the canvas is rotated and set to a default position.